docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -89,7 +89,9 @@ Nuxt 3 starting from rc.12 comes with the public beta for route rules and hybrid
 - `ssr` - Disables server-side rendering for sections of your app and make them SPA-only with `ssr: false`
 - `cors` - Automatically adds cors headers with `cors: true` - you can customize the output by overriding with `headers`
 - `headers` - Add specific headers to sections of your site - for example, your assets
-- `static` and `swr` - `static` enables a single (on-demand) build; `swr` enables a static build, that lasts for a configurable TTL. (currently enables full incremental static generation on Netlify, with Vercel coming soon)
+- `swr` - Server-side renders sections of your app and add cache headers in the response for the server or the reverse proxy to cache it for a configurable TTL. The node-server preset of Nitro is able to cache the response. For Netlify and Vercel, the response is also added to the CDN layer.
+- `static` - The behavior is the same as `swr` except that there is no TTL: the response is cached until the next deployment. On Netlify, it enables full incremental static generation. Vercel is coming soon.
+- `prerender` - Fetches your app sections and copies them as static assets during the build
```

Comment:
```suggestion
- `prerender` - Fetches your app routes and copies them as static assets during the build
```

Reply:
What is the result of using the static property when using Node instead of Vercel/Netlify? It's not clear to me what the static property does by reading the current and proposed documentation.